templates/data-sources/server.md.tmpl

```diff
@@ -36,8 +36,16 @@ This resource is useful if you want to use a non-terraform managed server.
 - `ipv6_network` - (string) The IPv6 network.
 - `status` - (string) The status of the server.
 - `labels` - (map) User-defined labels (key-value pairs)
+- `network` - (map) Private Network the server is attached to.
 - `firewall_ids` - (Optional, list) Firewall IDs the server is attached to.
 - `placement_group_id` - (Optional, string) Placement Group ID the server is assigned to.
 - `delete_protection` - (bool) Whether delete protection is enabled.
 - `rebuild_protection` - (bool) Whether rebuild protection is enabled.
 - `primary_disk_size` - (int) The size of the primary disk in GB.
+
+a single entry in `network` support the following fields:
+
+- `network_id` - (Required, int) ID of the network
+- `ip` - (Optional, string) Specify the IP the server should get in the network
+- `alias_ips` - (Optional, list) Alias IPs the server should have in the Network.
+- `mac_address` - (Optional, string) The MAC address the private interface of the server has
```

Comment:
Users can not set these fields, so "[field] the server should get/have in the network" does not really make sense. Please update it to indicate that this is what the server **has**.